backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/profiles/DiskProfileHelper.java

```diff
@@ -63,6 +64,9 @@ public ValidationResult setAndValidateDiskProfiles(Map<DiskImage, Guid> map, DbU
                         diskImage.getDiskStorageType(), diskImage.getDiskAlias());
                 continue;
             }
+            if (shouldSetDiskProfileOnImage(diskImage)) {
```

Comment:
```suggestion
            if (!shouldSetDiskProfileOnImage(diskImage)) {
```

Reply:
how about `shouldNotSetDiskProfileOnImage` ?

Reply:
also possible, although I prefer to be more positive than negative ;)